electrumsv/wallet_support/late_header_worker.py

```diff
@@ -88,12 +104,14 @@ async def late_header_worker_async(db_functions_async: AsynchronousFunctions,
 
 async def _populate_initial_state(db_functions_async: AsynchronousFunctions,
         state: LateHeaderWorkerState) -> None:
+    """On initial startup, the global Network singleton will sync the longest chain. This happens
```

Comment:
This comment is focuses on transactions without present headers. This function does not solely queue those, but all the ones in the database that need processing.

Reply:
Not sure I understand. `read_pending_header_transactions_async` gets all the transactions that are `STATE_CLEARED` with proof data and it only runs once when the wallet is loaded right?

So my point is that when initially loading up a wallet, it's hightly likely that the `Network` singleton has synced a number of headers that were not present before... Therefore we should do a round of processing immediately.

Reply:
Anyway. Clearly you have a problem with this comment. I will delete it and let the code speak for itself.